src/Microsoft.SqlTools.Serialization/Serialization.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.SqlTools.Hosting.Hosting.Contracts;
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.Serialization
```

Comment:
Probably best for consistency with the other code if we keep this in a Contracts namespace